java-squid/src/main/java/org/sonar/java/cfg/CFG.java

```diff
@@ -131,9 +137,63 @@ public Block(int id) {
     public Tree terminator() {
       return terminator;
     }
+
+    public boolean isActive() {
+      return terminator == null && elements.isEmpty();
+    }
+
+    @Override
+    public String toString() {
+      StringOutputStream buffer = new StringOutputStream();
+      debugTo(new PrintStream(buffer));
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Remove this use of constructor "PrintStream(OutputStream)" [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1943)